CMakeLists.txt

```diff
@@ -81,6 +81,7 @@ include_directories(${PROJECT_BINARY_DIR}/include)
 
 include_directories(include)
 
+install(DIRECTORY cub/ DESTINATION cub PATTERN .* EXCLUDE)
```

Comment:
I'm not sure if we should be doing this, or depending on cub as a third-party library. Let's talk about it in the dev meeting. It raises the larger issue of how we should handle dependencies.